db/migrate/20150902171923_add_features_tickets_and_acceptance_criteria.rb

```diff
@@ -0,0 +1,25 @@
+class AddFeaturesTicketsAndAcceptanceCriteria < ActiveRecord::Migration
+  def change
```

Comment:
Actually forget splitting them. Just double check we have all the classes and tables you think we need.